mobile/src/src/utils/endpoints.js

```diff
@@ -165,6 +240,30 @@ class BasePostingEndpoint extends BaseEndpoint {
 
 // Base comment endpoint class
 class BaseCommentEndpoint extends BaseEndpoint {
+  static contentType = null;
+
+  // objectId is nullable, you can omit it to get all comments if you need it for some reason
+  static list(
+    objectId, // <- The id of the post/listing/ad/etc that you're looking for the comments of
+    handleSuccess = () => {},
+    handleError = () => {}
+  ) {
+    Axios.get(
+      BASE_URL +
+        "?" +
+        "content_type=" +
+        this.contentType.toString() +
+        "&object_id=" +
+        objectId.toString()
+    )
```

Comment:
We should find a way to keep the original functionality to filter and add on these 2 new filters. Checkout BaseEndpoint.list() to see how it's implemented. We can just take on "content_type" and "object_id" filters near where the "page" filter is added